app/views/data_centers/_asn_list.html.erb

```diff
@@ -0,0 +1,16 @@
+<% @results[:results].each do |_, asn_data| %>
+  <tr>
+    <td><%= link_to asn_data[:asn], asn_url(network: params[:network], asn:asn_data[:asn]) %></td>
+    <td>
+      <% asn_data[:data_centers].each do |dc| %>
+        <%= link_to dc, data_center_url(network: params[:network], key: dc.gsub('/', '-slash-')) %>
+        <br \>
+      <% end %>
+    </td>
+    <td><%= asn_data[:count] %></td>
+    <td>
+      <%= number_with_delimiter lamports_to_sol(asn_data[:active_stake]).round(0) %>
+      (<%= number_to_percentage((asn_data[:active_stake] / @results[:total_stake].to_f) * 100.0, precision: 2) %>)
```

Comment:
This calculation `(asn_data[:active_stake] / @results[:total_stake].to_f) * 100.0` also could be in a helper.